eslint/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "jest",
     "clean": "rimraf dist",
     "build": "npm run clean && tsc --project tsconfig.json",
-    "version": "node ../build/bump.js"
+    "version": "node ../build/bump.js $npm_package_version"
```

Comment:
You don't need env $npm_package_version here, because
1. it is already `process.env` while mentioning it here produces problems on windows machines (out of cross-env runner)
2. action in version hook is executed after a native npm version process so actual version is also presented inside current package.json